### DISCUSSION TITLE
v3: How do you cancel file upload?

### DISCUSSION THREAD

**Discussion by smares** [livewire/livewire#6004] [1. Help]
With v3, there still doesn't seem to be a way to cancel file uploads, or am I wrong? How do you guys do it? With v2, we had to make our own fork of Livewire based on https://github.com/livewire/livewire/pull/3216.

**Comment by MasterDeveloper1989** [comment]
I have the same problem when canceling video upload. When I use the cancelUpload method in the LiveWire documentation, I get the following error: 
Unable to call component method. Public method [$cancelUpload] not found on component

**Comment by BVFI** [comment]
[Cancelling an upload](https://livewire.laravel.com/docs/uploads#cancelling-an-upload) can be done with $wire.cancelUpload() and it works fine. However, if you are talking cancelling this programatically and not waiting on the user to click on a cancel button, I don't think there is a way, or I havent been able to find one, besides making it myself. Regarding @MasterDeveloper1989's issue, I think you are calling it wrong, if you showed us some code, we might figure it out as Im sure its just a syntax error or sync issues.
